src/test/groovy/com/jaredsburrows/license/LicensePluginJavaSpec.groovy

```diff
@@ -724,23 +744,26 @@ final class LicensePluginJavaSpec extends Specification {
       """
       <html>
         <head>
-          <style>
-            body { font-family: sans-serif } 
-            pre { background-color: #eeeeee; padding: 1em; white-space: pre-wrap; display: inline-block }
-          </style>
+          <style>body { font-family: sans-serif } pre { background-color: #eeeeee; padding: 1em; white-space: pre-wrap; display: inline-block }</style>
           <title>Open source licenses</title>
         </head>
         <body>
           <h3>Notice for packages:</h3>
           <ul>
-            <li>
-              <a href="#314129783">appcompat-v7</a>
+            <li><a href="#1934118923">appcompat-v7 (26.1.0)</a>
```

Comment:
Sorry, one last thing before we merge, can we remove the parentheses around the version?

Reply:
I did that explicitly so that if there wasn't a version (theoretically possible, but, granted, unlikely) there's be an obvious placeholder that could be edited. I don't mind making the change, but there was a reason for doing it this way. If you still want a change, would you prefer no placeholder or some other placeholder?